crates/bevy_core_pipeline/src/core_2d/main_opaque_pass_2d_node.rs

```diff
@@ -65,14 +65,14 @@ impl ViewNode for MainOpaquePass2dNode {
 
             // Render pass setup
             let render_pass = command_encoder.begin_render_pass(&RenderPassDescriptor {
-                label: Some("main_opaque_pass_2d"),
+                label: Some("main_opaque_2d_pass"),
```

Comment:
I would rather keep the original main_opaque_pass_2d name, to match the node/file name.